### PR TITLE
Feature/ssh agent

### PR DESCRIPTION
**Pull request by mateus-amarante**
### Description

Share ssh keys with ssh-agent forwarding rather than sharing the ssh key files directly. It is accomplished by sharing the ssh-agent socket. The user just needs to add the private key with `ssh-add YOUR_PRIVATE_KEY_PATH`.

### Related Issues:
- #38

### CODE REVIEW

**Comment by mateus-amarante** [comment]
Friendly ping

File: run-devel.sh [hunk 1]
```diff
@@ -50,6 +50,8 @@ fi
 ROS_DISTRO=$ROS_DISTRO \
   VOLUMES_FOLDER=$VOLUMES_FOLDER \
   CONTAINER_NAME=$CONTAINER_NAME \
+  SSH_AUTH_SOCK_HOST_PATH="$SSH_AUTH_SOCK" \
+  SSH_AUTH_SOCK_CONTAINER_PATH="/home/orise/.ssh-agent/ssh-agent.sock" \
```
Comment (caioaamaral) [inline run-devel.sh:54]: shouldn't we guarantee that `ssh-agent` is using this specific socket?

Comment (mateus-amarante) [reply run-devel.sh:54]: This is the socket inside the container

Comment (mateus-amarante) [reply run-devel.sh:54]: The socket in the host is defined by `$SSH_AUTH_SOCK_HOST_PATH="$SSH_AUTH_SOCK`

**Review by caioaamaral — APPROVED** [review]
LGTM!

Just a question, should we add instructions regarding how to start ssh-agent and add new ssh keys?

**Comment by mateus-amarante** [comment]
> Just a question, should we add instructions regarding how to start ssh-agent and add new ssh keys?

Added in 5b6a3b1. Thanks for the review @caioaamaral

**Pull request merged**